src/main/resources/logback-spring.xml

```diff
@@ -0,0 +1,27 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
I propose to have logback.groovy instead of XML, XML is shit!

Reply:
Nice that you're using spring boot integration for logging, let's stick to it. Here's a page, we need to add groovy dependency and migrate logback-spring.xml to logback.groovy

Here's a link: https://docs.spring.io/spring-boot/docs/current/reference/html/howto-logging.html#howto-configure-logback-for-logging

Reply:
Great idea! I totally support that.

Reply:
You're my star!